src/lib/include/libsed.h

```diff
@@ -92,6 +94,21 @@ struct sed_key {
 	uint8_t len;
 };
 
+struct sed_opal_lockingrange {
+	size_t start;
+	size_t length;
+	uint8_t lr_id;
+	uint8_t read_locked;
```

Comment:
read_locked, write_locked, rle, wle AFAIK are single bits. we could space optimize here a little bit ....

Reply:
Done